libraries/radwordsprocessing/editing/find-and-replace.md

```diff
@@ -1,44 +1,67 @@
 ---
-title: Replace Text and Style
-page_title: Replace Text and Style
-description: Learn how to replace text or styling with WordsProcessing
+title: Find and Replace Text and Style
+page_title: Find and Replace Text and Style
+description: Learn how to find and replace text or styling with WordsProcessing
```

Comment:
Learn how to find and replace text content or styling in a document using WordsProcessing.